File: 001-two-sum.py
from typing import List


class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        d = {}
        for i in range(len(numbers)):
            if (target - numbers[i]) in d:
                return [d[target - numbers[i]] + 1, i + 1]
            d[numbers[i]] = i

File: 001-two-sum.js
var twoSum = function(nums, target) {
  prevSet = new Map();
  index1 = -1;
  index2 = -1;
  for (let i = 0; i < numet.has(target - nums[i])) {
    index2 = i;
    index1 = prevSet.get(target - nums[i]);
    return [index1, index2];
  }
  prevSet.set(nums[i], i);
}
};
